go.mod

```diff
@@ -151,5 +151,3 @@ require (
 // Workaround for indirect dependency no longer being available.
 // https://github.com/containerd/containerd/issues/9969
 exclude github.com/mitchellh/osext v0.0.0-20151018003038-5e2d6d41470f
-
-replace github.com/containerd/containerd/api => ./api
```

Comment:
Note that this means that the code in the `api` directory will no longer be tested. Currently that's fine, because the code in that directory is what's tagged as `api/v1.7.19`, but if changes are made after this, they will be untested (and the code is all dead-code).

If we don't plan to do maintenance on  api v1.7.x, then we should consider deleting the directory.